core/src/test/scala/com/pingcap/tispark/hostmap/HostMappingSuite.scala

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright 2022 PingCAP, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.pingcap.tispark.hostmap
+
+import com.pingcap.tikv.TiConfiguration
+import com.pingcap.tikv.hostmap.UriHostMapping
+import com.pingcap.tispark.TiConfigConst.PD_ADDRESSES
+import com.pingcap.tispark.utils.TiUtil
+import org.apache.spark.sql.SparkSession
+import org.apache.spark.{SparkConf, SparkContext}
+import org.apache.spark.sql.catalyst.catalog.TiCatalog
+import org.scalatest.{FunSuite, Matchers}
+import org.scalatest.Matchers.{be, convertToAnyShouldWrapper, have, noException, the}
+
+class HostMappingSuite extends FunSuite {
+  val TIKV_HOST_MAPPING: String = "spark.tispark.tikv.host_mapping"
+
+  test("Host map configuration test") {
+    val conf: SparkConf = new SparkConf(false)
+    conf.set(TIKV_HOST_MAPPING, "host1:1;host2:2;host3:3")
+    val tiConf: TiConfiguration = new TiConfiguration()
+    TiUtil.sparkConfToTiConfWithoutPD(conf, tiConf)
+    assert(tiConf.getHostMapping.asInstanceOf[UriHostMapping].getHostMapping.get("host1") == "1")
+    assert(tiConf.getHostMapping.asInstanceOf[UriHostMapping].getHostMapping.size() == 3)
+  }
+
+  test("Host map configuration test with wrong host map") {
+    the[IllegalArgumentException] thrownBy {
+      val conf: SparkConf = new SparkConf(false)
+      conf.set(TIKV_HOST_MAPPING, "")
```

Comment:
Empty is reasonably considered to be valid

Reply:
OK, I will change the logic here.